### DISCUSSION TITLE
Now introducing custom CSS!

### DISCUSSION THREAD

**Discussion by qweliant** [knowledgefutures/pubpub#1559] [Show and tell]
Admins now have the ability to add custom css to their communities via a custom scripts editor

![image](https://user-images.githubusercontent.com/34730449/132751833-bccfdd5f-c494-4e52-a0a1-144d6e83836c.png)

![image](https://user-images.githubusercontent.com/34730449/132751878-45926f69-5144-4029-ae0f-66167b58aab5.png)

**Comment by jeffpooley** [comment]
Profoundly excited about this!! Thank you!

**Reply by qweliant** [reply]
No problem! Many thanks to the KFG for making it happen! Have fun!

**Reply by prayertitus** [reply]
Nice Update
